src/GT.UI/Views/Listing/Listing.cshtml

```diff
@@ -1,5 +1,24 @@
-﻿@*
-    For more information on enabling MVC for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
-*@
+﻿@model GT.Core.DTO.Impl.ListingDTO
 @{
+	ViewData["Title"] = "Annons";
+
 }
+
+<h1>Annons</h1>
+
+<h2>@Model.ListingTitle</h2>
+<div class="ListingData">
```

Comment:
Shouldn't HTML class names generally be lower case?

Reply:
Fixed